setup.py

```diff
@@ -31,8 +33,8 @@ def get_version(*file_paths):
     os.system("git push --tags")
     sys.exit()
 
-README = open(os.path.join(os.path.dirname(__file__), 'README.rst')).read()
-CHANGELOG = open(os.path.join(os.path.dirname(__file__), 'CHANGELOG.rst')).read()
+README = io.open(os.path.join(os.path.dirname(__file__), 'README.rst')).read()
+CHANGELOG = io.open(os.path.join(os.path.dirname(__file__), 'CHANGELOG.rst')).read()
```

Comment:
The [`url` given below](https://github.com/open-craft/openedx-completion-aggregator/pull/12/files#diff-2eeaed663bd0d25b7e608891384b7298R46) points to the `edx` org instead of `open-craft`.

Need to update these as well:
* `setup.py`: `author` and `author-email`
* `README.rst`: the CI links,  security alert email address